about_me.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles/about_me.css">
+    <link rel="stylesheet" href="normalize.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300"
+      rel="stylesheet">
+    <title>About Chantal</title>
+  </head>
+
+  <body>
+    <img alt="foggy blue mountains" src="assets/background.jpg" />
+    <header>
+      <nav class="site-nav">
+        <a href="about_me.html">About Me</a>
+        <a href="portfolio.html">Portfolio</a>
+        <a href="hobby_blog.html">Hobby-Blog</a>
+        <a href="index.html">Home</a>
+      </nav>
+    </header>
+
+    <section>
```

Comment:
Think about the meaning behind section, article and div.  I would probably call this section or article "about-me" and omit the div... I've added another comment about these tags below with a link to a helpful blog post about these three elements.